run/logging-manual/test/system.test.js

```diff
@@ -144,18 +148,26 @@ describe('Logging', () => {
       // The latest two log entries for our service in the last 5 minutes
       // are treated as the result of the previous test.
       // Concurrency is supporting by distinctly named service deployment per test run.
-      const filter = `resource.labels.service_name="${service_name}" timestamp>="${dateMinutesAgo(
-        new Date(),
-        5
-      )}"`;
-      const entries = await getLogEntriesPolling(filter);
-      entries.forEach(entry => {
-        if (entry.metadata.httpRequest) {
-          requestLog = entry;
-        } else {
-          sampleLog = entry;
-        }
-      });
+      let entries;
+      let attempt = 0;
+      const maxAttempts = 5;
+      while ((!requestLog || !sampleLog) && attempt < maxAttempts) {
+        await sleep(attempt * 30000); // Linear backoff between retry attempts
```

Comment:
Future: It seems like the retry logic is different in each repo. No objection there for now, but something to flag in case we notice different levels of flakiness in the future.